<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ydpd</class>
 <widget class="QWidget" name="Ydpd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ydpd</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="toolbarLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>13</number>
     </property>
     <item>
      <widget class="QToolButton" name="prevToolButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Poprzednie tłumaczenie</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="Ydpd.qrc">
         <normaloff>:/res/arrow-left.png</normaloff>:/res/arrow-left.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+Shift+,</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="nextToolButton">
       <property name="toolTip">
        <string>Kolejne tłumaczenie</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="Ydpd.qrc">
         <normaloff>:/res/arrow-right.png</normaloff>:/res/arrow-right.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+Shift+.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="playToolButton">
       <property name="toolTip">
        <string>Odtwórz próbkę dźwiękową</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="Ydpd.qrc">
         <normaloff>:/res/text-speak.png</normaloff>:/res/text-speak.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+P</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="optionsToolButton">
       <property name="toolTip">
        <string>Konfiguracja</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="Ydpd.qrc">
         <normaloff>:/res/configure.png</normaloff>:/res/configure.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+O</string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::MenuButtonPopup</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="foreignRadioButton">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="shortcut">
        <string>F1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="nativeRadioButton">
       <property name="toolTip">
        <string>Przełącz na polski</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Ydpd.qrc">
         <normaloff>:/res/pl.png</normaloff>:/res/pl.png</iconset>
       </property>
       <property name="shortcut">
        <string>F2</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="dictionaryComboBox">
       <property name="toolTip">
        <string>Zmień słownik</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="opaqueResize">
      <bool>true</bool>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="splitterLayoutWidget">
      <layout class="QVBoxLayout" name="leftLayout" stretch="0,0">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="historyComboBox">
         <property name="toolTip">
          <string>Historia</string>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="insertPolicy">
          <enum>QComboBox::NoInsert</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToMinimumContentsLength</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="entriesListView">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="focusPolicy">
          <enum>Qt::WheelFocus</enum>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="uniformItemSizes">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTextEdit" name="descriptionTextEdit">
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>prevToolButton</tabstop>
  <tabstop>nextToolButton</tabstop>
  <tabstop>historyComboBox</tabstop>
  <tabstop>entriesListView</tabstop>
  <tabstop>descriptionTextEdit</tabstop>
 </tabstops>
 <resources>
  <include location="Ydpd.qrc"/>
 </resources>
 <connections/>
</ui>
